src/main/kotlin/com/game/catbackend/domain/services/ImageService.kt

```diff
@@ -14,6 +14,9 @@ class ImageService(val imageRepository: ImageRepository) {
         val image = imageDto.toImage()
         image.active = true
 
+        if (image.source.length > 200) {
+            return throw Exception("The len of image source needs to be less than 200 characters!")
```

Comment:
Create a more especific exception for this error. Something among the lines of ImageInvalidAttributeException